server/src/main/java/io/crate/planner/operators/LogicalPlanner.java

```diff
@@ -156,9 +156,6 @@ public class LogicalPlanner {
         new RewriteToQueryThenFetch()
     );
 
-    public static final List<Rule<?>> WRITE_OPTIMIZER_RULES =
-        List.of(new RewriteInsertFromSubQueryToInsertFromValues());
-
```

Comment:
I'd vote for keeping the static var and add a comment with a short explanation why this is not used as a configurable optimizer rule.

Reply:
👍